docs/shared/react-standalone-tutorial/1-code-generation.md

```diff
@@ -0,0 +1,187 @@
+---
+title: 'React Standalone Tutorial - Part 1: Code Generation'
+description: In this tutorial you'll create a frontend-focused workspace with Nx.
+---
+
+{% callout type="check" title="Standalone Repo" %}
+This tutorial sets up a repo with one main application that breaks out its code into libraries to add structure.
+
+You can find more information on about other repo styles in [our introduction](/getting-started/intro).
+{% /callout %}
+
+# React Tutorial - Part 1: Code Generation
+
+## Contents
+
+- [1 - Code Generation](/react-standalone-tutorial/1-code-generation)
+- [2 - Project Graph](/react-standalone-tutorial/2-project-graph)
+- [3 - Task Running](/react-standalone-tutorial/3-task-running)
+- [4 - Workspace Optimization](/react-standalone-tutorial/4-workspace-optimization)
+- [5 - Summary](/react-standalone-tutorial/5-summary)
+
+## Your Objective
```

Comment:
Causing is incosistent. Sometimes it's title case, other times it isn't. We should make the same as the other tutorials